public/app/features/dashboard/specs/DashboardRow.jest.tsx

```diff
@@ -42,7 +36,7 @@ describe('DashboardRow', () => {
   });
 
   it('should have zero actions as viewer', () => {
-    config.bootData.user.orgRole = 'Viewer';
+    dashboardMock.meta.canEdit = false;
```

Comment:
Naming of test should be changed to reflect possibility to edit instead of having role viewer.